radiation/python/radsw/old/taumol.py

```diff
@@ -75,9 +75,14 @@
 
 indict = dict()
 for var in invars:
-    indict[var] = serializer.read(
-        var, serializer.savepoint["swrad-taumol-input-000000"]
-    )
+    if var == "NLAY":
+        indict[var] = serializer.read(
+            var, serializer.savepoint["swrad-taumol-input-000000"]
+        )[0]
+    else:
+        indict[var] = serializer.read(
+            var, serializer.savepoint["swrad-taumol-input-000000"]
+        )[10, ...]
```

Comment:
ah this was because of the strange way `idxday` works. index [0] of `idxday` is 9, so the 9th element of each array will have values, but indices 1-8 won't, so I had to select an index that has data to use as input, I just chose 10 randomly, others would work.